.github/workflows/main.yml

```diff
@@ -0,0 +1,37 @@
+name: Build and Deploy Docs to Netlify
```

Comment:
If we go this route, I would probably just push the documentation to gh-pages or perhaps readthedocs.

Reply:
I will setup pushthedocs a bit later then